### PR TITLE
Feature/8371 qa landing page

### PR DESCRIPTION
**Pull request by ninjagirl888**
#### 📲 What

QA Landing page created
Ensono Stacks Homepage added button to link to QA Landing page
Deprecated Cypress information removed
.NET Selenium removed.

#### 🤔 Why
		
To enable users to see an overview of what testing there is offered by Ensono Stacks solution.
Redundant information should not be available as to not confuse users.
		
#### 🛠 How
		
Documentation update.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.
		 
#### 🕵️ How to test

Clone repos
and run locally
npm run build
npm run start

#### ✅ Acceptance criteria Checklist

- [x] Code peer reviewed?
- [x] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [x] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: docs/testing/testing_overview.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- **Accessibility Testing**: [Accessibility Testing with Playwright](./testing_in_nx/playwright_accessibility_testing.md)
+- **Applitools Integration**: [Playwright with Applitools Eyes](./testing_in_nx/playwright_visual_testing_applitools.md)
+
+### API Testing
+
+#### Ensono Stacks .NET
+The Ensono Stacks .NET includes a functional testing framework using xUnit and BDDfy. This framework allows you to write and run tests to ensure that your API behaves correctly. For more details, refer to the [Functionally Testing the API](../workloads/azure/backend/netcore/testing/functional_testing_netcore.md) documentation.
+
+#### Ensono Stacks JAVA
+The Ensono Stacks JAVA includes an API testing framework using Serenity BDD and Rest Assured. This framework allows you to write and run tests to ensure that your API behaves correctly. For more details, refer to the [Running API tests with Serenity](../workloads/azure/backend/java/testing/execute_serenity_api_tests.md) documentation.
```
Comment (JackMidd-Amido) [inline docs/testing/testing_overview.md:51]: Java doesn't need to be uppercase. Java is fine :)

File: docs/testing/testing_overview.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+Contract testing ensures that the interactions between services are working as expected. Ensono Stacks uses [Pact](https://docs.pact.io/) for consumer-driven contract testing.  For further information please refer to the [API Contract Testing with Pact](./contract_testing_pact.md) documentation.
+
+### ServiceBus Testing
+Ensono Stacks .NET includes comprehensive functional and unit tests for ServiceBus operations to ensure the correct handling of topics, queues, and messages. Tests coverage include 
+ - Adding messages to the queue
+ - Checking the existence of topics
+ - Reading messages from the queue
+ - Clearing messages from the queue
+
+Details of available tests can be found in the [Functions and Worker Services](../workloads/azure/backend/netcore/architecture/project_structure/functions_and_worker_netcore.md) documentation.
```
Comment (JackMidd-Amido) [inline docs/testing/testing_overview.md:64]: The page linked does not contain anything about functional testing. I think we should point users to the functional testing folder in the actual repo as the repo itself is public.

This folder contains the functional tests covering Service Bus Testing.
Well, it is a project that tests:
- Adding/Updating/Deleting items in cosmos which triggers a function to notify a Service bus instance of the Cosmos event
- Checks the service bus topics for relevant messages

You should update this section and maybe the cosmos section to talk about this and the unit/integration testing that are already being mentioned.

File: docs/testing/testing_overview.md (hunk 1 above)
Comment (ninjagirl888) [reply docs/testing/testing_overview.md:51]: updated to Java

File: docs/testing/testing_overview.md (hunk 2 above)
Comment (ninjagirl888) [reply docs/testing/testing_overview.md:64]: updated with links to repository.

**Review by JackMidd-Amido — APPROVED** [review]

**Pull request merged**